manifests/node-exporter-daemonset.yaml

```diff
@@ -3,6 +3,8 @@ kind: DaemonSet
 metadata:
   labels:
     app: node-exporter
```

Comment:
this `app` label shouldn't be here anymore now I think

Reply:
Nice catch. Fixed it and resolved conflicts.